server/models/Module.py

```diff
@@ -28,13 +28,16 @@ class Meta:
     author = models.CharField('author', max_length=200, default="Workbench")
 
     # the url of the module repo
-    link = models.CharField('link', max_length=200, default="")
+    link = models.URLField('link', max_length=200, default="")
 
     # icon name associated with module
     icon = models.CharField('icon', max_length=20, default='url')
 
     # Does this model bring in external data?
     loads_data = models.BooleanField('loads_data', default=False)
 
+    # URL for the module's documentation, defaults to our knowledge base root
+    help_url = models.URLField('help_url', max_length=200, default=False)
```

Comment:
What is default=False, and why False here and '' for link field?
Also, I suspect we want only partial (relative)  URLS for the help_url (see below), will URLField still handle that?